lib/Conversion/GPUToSPIRV/GPUToSPIRVPass.cpp

```diff
@@ -0,0 +1,122 @@
+//===- GPUToSPIRVPass.cpp -  --------------*- C++ -*-===//
+//
+// Copyright 2022 Intel Corporation
+// Part of the IMEX Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file extends upstream GPUToSPIRV Pass that converts GPU ops to SPIR-V
+/// by adding more conversion patterns like SCF, math and control flow. This
+/// pass only converts gpu.func ops inside gpu.module op.
+///
+//===----------------------------------------------------------------------===//
+#include "imex/Conversion/GPUToSPIRV/GPUToSPIRVPass.h"
+
+#include "../PassDetail.h"
+#include "mlir/Conversion/MemRefToSPIRV/MemRefToSPIRV.h"
+#include "mlir/Dialect/GPU/IR/GPUDialect.h"
+
+#include <mlir/Conversion/ArithmeticToSPIRV/ArithmeticToSPIRV.h>
```

Comment:
I think some of those includes can be removed.

Reply:
Done